src/crypto-api.ts

```diff
@@ -229,6 +229,17 @@ export interface CryptoApi {
      */
     createRecoveryKeyFromPassphrase(password?: string): Promise<GeneratedSecretStorageKey>;
 
+    /**
+     * Check the cross signing trust of the current user
```

Comment:
> Check the cross signing trust of the current user

what happens if the check fails?

Reply:
Depends when the failure occurs; the device will not be verified (the device signatures will not be uploaded), the cross signing keys or the device keys will no be imported

Reply:
please put this in the documentation

Reply:
(will an exception be thrown if the check fails?)